reference/commands/new.rst

```diff
@@ -30,6 +30,14 @@ package testing files.
      -b, --bare     Create the minimum package recipe, without build() or
                     package()methods. Useful in combination with "package_files"
                     command
+     -cis, --ci_shared        Package will have a "shared" option to be used in CI
```

Comment:
Few lines below, the `$ conan new` command should be:

` conan new [-h] [-t] [-i] [-c] [-s] [-b] [-cis] [-cilg] [-cilc] [-cio]
                 [-ciw] [-gi] [-ciu CI_UPLOAD_URL]`